images/nats/image.yaml

```diff
@@ -8,3 +8,10 @@ versions:
         - suffix: -dev
           options:
             - dev
+options:
+  dev:
```

Comment:
lets call this "nats-dev", then add it the the list above under dev, otherwise I think it will override the global dev which has apk-tools etc

Reply:
Good call!